cpu/efr32mg1p/ldscripts/efr32mg1p132f256gm32.ld

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2015-2017 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @addtogroup      cpu_efr32mg1p
+ * @{
+ *
+ * @file
+ * @brief           Memory definitions for the EFR32MG1P132F256GM32 CPU
+ *
+ * @author          Hauke Petersen <hauke.petersen@fu-berlin.de>
+ * @author          Bas Stottelaar <basstottelaar@gmail.com>
+ *
+ * @}
+ */
+
+MEMORY
+{
+    rom (rx)    : ORIGIN = 0x00000000, LENGTH = 262144
+    ram (rwx)   : ORIGIN = 0x20000000, LENGTH = 31744
```

Comment:
What happened with the remaining 1K of RAM?

Reply:
If I recall correctly, 1 KiB is reserved for the radio. I've [double-checked](https://github.com/SiliconLabs/Gecko_SDK/blob/master/platform/Device/SiliconLabs/EFR32MG1P/Source/GCC/efr32mg1p.ld) with the Gecko SDK, but that number is correct :-)

Reply:
Ok. I suppose that is managed internally since I don't see any specific instruction in the ld you cite on which they make use of the last 1K of RAM. Though it looks a bit strange but if it works let's continue this way.